src/sandstorm/supervisor.capnp

```diff
@@ -64,9 +64,6 @@ interface SandstormCore {
   # When the supervisor receives a save() request for a capability hosted by the app, it first
   # calls save() on the underlying capability to get an AppSturdyRef, then calls wrapSaved() to
   # convert this to a token which it can then return.
-  #
-  # TODO(soon): How do we keep this capability associated with the user account that created it,
```

Comment:
Presumably this is what you're trying to solve with adding `userId` to `SturdyRefOwner`.

I think the right solution to this issue is actually that when a capability comes out of the Powerbox, it is automatically wrapped in a revokable membrane that is revoked if the user who performed the Powerbox interaction loses access. The later `save()` call on that powerbox capability goes through this membrane which appropriately annotates the resulting SturdyRef with this information. The membrane could possibly be implemented by the supervisor for the app which hosts the capability, in order to avoid network hops through any other intermediates.